packages/symfony-static-dumper/tests/Application/SymfonyStaticDumperApplicationTest.php

```diff
@@ -89,5 +89,33 @@ public function test(): void
             self::OUTPUT_DIRECTORY . '/static.html',
             self::OUTPUT_DIRECTORY . '/static/index.html'
         );
+
+        // controller with 1 arg
+        $this->assertFileExists(self::OUTPUT_DIRECTORY . '/one-param/1/index.html');
+        $this->assertFileExists(self::OUTPUT_DIRECTORY . '/one-param/2/index.html');
+
+        $this->assertFileEquals(
+            self::EXPECTED_DIRECTORY . '/one-param/1.html',
+            self::OUTPUT_DIRECTORY . '/one-param/1/index.html'
+        );
+        $this->assertFileEquals(
+            self::EXPECTED_DIRECTORY . '/one-param/2.html',
+            self::OUTPUT_DIRECTORY . '/one-param/2/index.html'
+        );
+
+        // Controller with 2 args
+        $this->assertFileExists(self::OUTPUT_DIRECTORY . '/test/1/index.html');
+        $this->assertFileExists(self::OUTPUT_DIRECTORY . '/test/2/index.html');
+        $this->assertFileExists(self::OUTPUT_DIRECTORY . '/foo/1/index.html');
+        $this->assertFileExists(self::OUTPUT_DIRECTORY . '/foo/2/index.html');
+
+        foreach (['test', 'foo'] as $type) {
+            foreach ([1, 2] as $param) {
+                $this->assertFileEquals(
+                    self::EXPECTED_DIRECTORY . '/two-params/'. $type . $param . '.html',
+                    self::OUTPUT_DIRECTORY . sprintf('/%s/%s/index.html', $type, $param)
+                );
+            }
+        }
```

Comment:
This method is getting very fat.
New test cases should be at least in own method, better own test case class with clear naming.